pmm-templates/MongoDB_Overview.jsonnet

```diff
@@ -22,6 +22,9 @@ dashboard.new(
   tags=[ "MongoDB","Percona"],
   iteration=1553588305920,
   uid="6Lk9wMHik",
+  timepicker = timepicker.new(
+    hidden = true,
```

Comment:
Now it's hidden but it has to be shown.

Reply:
I don't understand what does that mean??

Reply:
The TimePicker element is hidden in your template.
![Screenshot_20190717_124057](https://user-images.githubusercontent.com/25184101/61365292-24f05d00-a890-11e9-9352-13f03b04e5ac.png)
It has to be shown.
e.g.
![Screenshot_20190717_123958](https://user-images.githubusercontent.com/25184101/61365213-038f7100-a890-11e9-8b00-d481426756da.png)